ecl/hql/hqlmanifest.cpp

```diff
@@ -221,9 +242,13 @@ void ResourceManifest::addToArchive(IPropertyTree *archive)
                 if (!additionalFiles->hasProp(xpath.str()))
                 {
                     IPropertyTree *resTree = additionalFiles->addPropTree("Resource", createPTree("Resource"));
+                    StringBuffer filepath(filename);
+                    if (item.queryProp("@repository"))
```

Comment:
I think this is a case where you do not want to fetch the artefact at this point.  This is before the query is sent to the system - so it would be downloaded by the user, added to the archive and then uploaded.  The aim of this change is to avoid that step.

Reply:
Ok, I think I understand. I am just not sure where the resource is used after being added to the archive.